src/sass/main.scss

```
@@ -165,3 +199,31 @@ figure {
     background: $alternative-background-color;
   }
 }
+
+.button {
+  font-weight: bold;
+  color: #f8f9fa;
```

```suggestion
  color: $gray-100;
```
I guess you mean this.